roaster-tm4j/roaster-tm4j-common/src/main/java/hu/icellmobilsoft/roaster/tm4j/common/client/RestTm4jService.java

```diff
@@ -0,0 +1,89 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.tm4j.common.client;
+
+import hu.icellmobilsoft.roaster.tm4j.dto.domain.test_execution.Execution;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.Response.Status.Family;
+import javax.ws.rs.core.Response.StatusType;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Class for handling the TM4J client calls
+ *
+ * @author martin.nagy
+ * @since 0.2.0
+ */
+@Dependent
+public class RestTm4jService {
+
+    @Inject
+    private Tm4jRestClient tm4jClient;
```

Comment:
Ide nem hianyzik a @RestClient annotacio?

Reply:
A `@RestClient` annotáció nem kötelező, anélkül is ugyan úgy működik. De azért ráraktam, hogy világosabb legyen.